CDCgov/NEDSS-Modernization
# Redirection Lib

## Description

To enable Page Builder and other services to implement a simple RedirectController, this PR creates a redirect library. This library has a single endpoint `/redirect` that accepts a `Location` header and issues a redirect to the specified location.

The intent is to force the trigger of the authentication Filters that will provide the `nbs_token` prior to loading a modernized page.

## Tickets

N/A

## Steps to verify

1. Build and run `nbs-gateway` (with page builder enabled) and `question-bank` services
2. Navigate to [local NBS[(http://localhost:8080/nbs/login) in incognito, or after clearing cookies
3. Navigate to System Management -> Page Management -> Manage Pages
4. Verify modernized page library is loaded as expected

Note -- this will require modifying the Helm chart to fully function in the `Int1` environment.
https://github.com/CDCgov/NEDSS-Helm/pull/173
